x-pack/plugins/siem/public/containers/hosts/first_last_seen/mock.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HostFirstLastSeenGqlQuery } from './first_last_seen.gql_query';
+
+interface MockedProvidedQuery {
+  request: object;
+  result: object;
```

Comment:
Optional, you could type these maybe with the GraphQL outputted types such as `HostFirstLastSeen`?

No changes requested though. Just a suggestion.